babel.config.cjs

```diff
@@ -49,6 +49,7 @@ module.exports = function (api) {
       'babel-plugin-styled-components',
       ['@babel/plugin-proposal-class-properties', { loose: true }],
       ['@babel/plugin-proposal-private-methods', { loose: true }],
+      ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
```

Comment:
Can you explain this change? I don't see how it's related to this.

Reply:
One of the dependencies updated and this line was needed.

Reply:
As per my other comments, it seems like those dependency updates were accidental